grpc/runtime/src/main/scala/io/buoyant/grpc/runtime/ServerDispatcher.scala

```diff
@@ -99,11 +103,12 @@ object ServerDispatcher {
         h2.Response(h2.Status.Ok, frames)
     }
 
-    private[this] def respondStreaming[Rsp](codec: Codec[Rsp], msgs: Stream[Rsp]): h2.Response = {
+    private[this] def respondStreaming[Rsp](codec: Codec[Rsp], msgs: Stream[Rsp], reqMeta: Seq[(String, String)]): h2.Response = {
       val frames = h2.Stream()
       def loop(): Future[Unit] =
         msgs.recv().transform {
           case Return(Stream.Releasable(s, release)) =>
+            log.trace(s"Streaming response metadata: ${reqMeta.mkString(":")}")
```

Comment:
This says response metadata but it's printing the request headers, right?

Reply:
What's the intent here?  Each time there's a response message, the request headers get printed?

Reply:
That's correct since the request headers may contain information useful for the particular request.